app/jobs/reports/sp_active_users_report.rb

```diff
@@ -4,7 +4,15 @@ module Reports
   class SpActiveUsersReport < BaseReport
     REPORT_NAME = 'sp-active-users-report'.freeze
 
-    def call
+    include GoodJob::ActiveJobExtensions::Concurrency
+
+    good_job_control_concurrency_with(
+      enqueue_limit: 1,
+      perform_limit: 1,
+      key: -> { "#{REPORT_NAME}-#{arguments.first}" },
```

Comment:
weird how some of these are not covered

Reply:
Do you think we should add explicit test coverage for them?

Reply:
It seems like we should already have it.

Reply:
I think these blocks only get called when jobs are enqueued to the DB, but a lot of our test coverage calls perform directly

Reply:
see d486cac2d2fe7d3545d5d8350000e6708bf678a4